.github/workflows/ci_cd.yml

```diff
@@ -110,12 +110,94 @@ jobs:
     name: "Check library vulnerabilities"
     runs-on: ubuntu-latest
     steps:
-      - uses: ansys/actions/check-vulnerabilities@v8.1
+      - uses: ansys/actions/check-vulnerabilities@feat/allow-for-extra-targets-in-vulnerabilities-checks
```

Comment:
TODO: revert to a released version

We need the ability to install extra targets for this check to work.